metadata-ingestion/tests/integration/powerbi/golden_test_container.json

```diff
@@ -498,13 +685,15 @@
     }
 },
 {
-    "entityType": "corpuser",
-    "entityUrn": "urn:li:corpuser:users.User1@foo.com",
```

Comment:
the corp user key is here. same as before.

Reply:
@hsheth2  this is for https://github.com/datahub-project/datahub/pull/7519/files/828fb8dd1dedd243957b6a9bd5aaf291a7bc1da9#r1163567554

There are no breaking changes.